CRM/Event/Page/ManageEvent.php

```diff
@@ -314,7 +314,11 @@ public function browse() {
     $this->_searchResult = CRM_Utils_Request::retrieve('searchResult', 'Boolean', $this);
 
     $whereClause = $this->whereClause($params, FALSE, $this->_force);
-    $this->pagerAToZ($whereClause, $params);
+
+    $config = CRM_Core_Config::singleton();
+    if ($config->includeAlphabeticalPager) {
```

Comment:
is it possible to simplify this to as we don't use $config at all anywhere else AFAIK in this function
```suggestion
    if (CRM_Core_Config::singleton()->includeAlphabeticalPager) {
```